docs/conf.py

```diff
@@ -77,6 +78,11 @@ def __getattr__(cls, name):
     'pint': ('https://pint.readthedocs.io/en/stable/', None),
     'cftime': ('https://unidata.github.io/cftime/', None),
 }
+# allows us to easily link PRs and issues in the change log
+extlinks = {
+    "issue": ("https://github.com/NCAR/geocat-comp/issues/%s", "GH#"),
+    "pr": ("https://github.com/NCAR/geocat-comp/pull/%s", "GH#"),
```

Comment:
```suggestion
    "issue": ("https://github.com/NCAR/geocat-comp/issues/%s", "GH"),
    "pr": ("https://github.com/NCAR/geocat-comp/pull/%s", "PR"),
```

Could we do this to distinguish between issues and pull requests? (This is what xarray does)

Reply:
Sure! I was just modeling this off of what Dask does for their release notes formatting. But a distinction is a good idea. Although, I'm not sure when an issue will ever make it into the release notes, but one could be added elsewhere

Reply:
Totally fair. I think it might be relevant to link to an issue if significant discussion happened on it, but yeah you're right it might not actually be used.

Reply:
It certainly doesn't hurt to add it, so I'll go ahead and commit this